### DISCUSSION TITLE
About locking the tool I chose

### DISCUSSION THREAD

**Discussion by sauching2002** [RyotaUshio/obsidian-pdf-plus#292] [Q&A]
I love this plugin so much, but I don’t know how to lock the tool I chose, especially the rectangle tool. I want to use the rectangle tool to excerpt content from PDFs with auto-paste continuously, and without having to toggle it every time I finish annotating. I hope there is a button to lock it.However, I’m not sure if there is a setting that could enable this?

**Comment by RyotaUshio** [comment]
I haven't implemented such an option yet. But there is a command called "Start rectangular selection" and you can assign a hotkey to it to speed up your workflow. 

I will consider adding a new option for that too.
